products/sle/main.pm

```diff
@@ -870,7 +870,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
     }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         load_virt_guest_install_tests;
-        if (is_x86_64 && !(get_var("GUEST_PATTERN") =~ /win/img) && !get_var("LTSS")) {
+        if (!(get_var('GUEST_PATTERN', '') =~ /win/img) && is_x86_64 && !get_var("LTSS")) {
```

Comment:
```suggestion
        if (!(get_var('GUEST_PATTERN') =~ /win/img) && is_x86_64 && !get_var('LTSS')) {
```
The `get_var` returns either the content of diver variable or `undef` (see [doc](http://open.qa/api/testapi/#_get_var)) and that should not be a problem in Perl.

Reply:
Have confirmed this suggestion as the above, but still get the same Perl warnings as below:
`Use of uninitialized value in pattern match (m//) at sle/tests/virt_autotest/install_package.pm line 103.`